velox/expression/VectorUdfTypeSystem.h

```diff
@@ -1104,6 +1105,10 @@ struct VectorReader<Generic> {
     return GenericView{base_, index};
   }
 
+  exec_null_free_in_t readNullFree(vector_size_t offset) const {
```

Comment:
Is this changed related to this PR ? On a quick look, doesn't seem like.

Reply:
@aditi-pandit This is an artifact of me stacking diffs in Phabricator, then exporting the top diff to GitHub. The PR ends up containing changes from 2 diffs in 2 commits.